make/libraries

```diff
@@ -113,9 +113,11 @@ TBB_CXX_TYPE ?= $(CXX_TYPE)
 # Set c compiler used for the TBB
 ifeq (clang,$(CXX_TYPE))
   TBB_CC ?= $(subst clang++,clang,$(CXX))
+  TBB_CPLUS_FLAGS ?= -Wno-unknown-warning-option -Wno-deprecated-copy
```

Comment:
Mind if this is renamed to something that's more consistent with our makefiles: `CXXFLAGS_TBB`?

If you want to have `TBB` out in front, then `TBB_CXXFLAGS`. Make's convention is for the C++ flags to be called `CXXFLAGS` and C flags (including preprocessor) to be `CFLAGS`. You can see in L147 that you're setting the `CXXFLAGS` variable.

Reply:
I would go with TBB_CXXFLAGS.

The cplus name was there becaus initiailly I tried with TBB’s CPLUS argument.

I prefer TBB as prefix because this flag is consumee by TBB makefiles not ours. To make a distinction.

Reply:
If I recall right, then the `CXXFLAGS_TBB` is used to tell our build process how the TBB is used in our builds (so when we use the TBB), but here it's about building the TBB itself - so we should distinguish as @rok-cesnovar did. Going with the TBB makefile conventions, which use CPLUS sounds logical to me as well.

Reply:
I think TBB_CXXFLAGS would be better because in the end I used the TBB's CXXFLAGS setting when calling make (I found this suggestion somewher on the TBB forums, the CPLUS stuff didnt work the way I expected).

So I think TBB_CXXFLAGS would be best, to distinguish from CXXFLAGS_TBB

Reply:
```cpp
"$(TBB_ABSOLUTE_PATH)/build/Makefile.tbb" compiler=$(TBB_CXX_TYPE) cfg=release stdver=c++1y  CXXFLAGS="$(TBB_CPLUS_FLAGS)"
```
but it will now be
```cpp
"$(TBB_ABSOLUTE_PATH)/build/Makefile.tbb" compiler=$(TBB_CXX_TYPE) cfg=release stdver=c++1y  CXXFLAGS="$(TBB_CXXFLAGS)"
```

Reply:
sounds good